src/Bugsnag.js

```diff
@@ -3,6 +3,9 @@ import { NativeModules } from 'react-native';
 const NativeClient = NativeModules.BugsnagReactNative;
 
 const BREADCRUMB_MAX_LENGTH = 30;
+const CONSOLE_LOG_METHODS = [ 'log', 'debug', 'info', 'warn', 'error' ].filter(method =>
+  typeof console[method] === 'function'
```

Comment:
it appeared (in the android emulator at least) that `console.debug()` does not exist. However, it does in Chrome/Node and so rather than fully hardcode this list I thought I'd list all of the console methods that we would want to wrap, and filter that list by which actual exist in the current environment.